doc/services/smf/index.rst

```diff
@@ -79,6 +117,13 @@ To transition from one state to another, the ``smf_set_state`` function is
 used and it has the following prototype:
 ``void smf_set_state(smf_ctx *ctx, smf_state *state)``
 
+**NOTE:** If :kconfig:option:`CONFIG_SMF_INITIAL_TRANSITION` is not set,
```

Comment:
There is a `.. note::` Sphinx directive that would probably be appropriate here and in other places with `**NOTE**`s